toolkit/tools/imagecustomizer/container/process-oci-artifact.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# entrypoint.sh
+
+# The first argument is expected to be the OCI artifact path.
```

Comment:
Lanze, we can capture the usage in the doc files as well (see [docs/cl](https://github.com/microsoft/azurelinux/tree/3.0-dev/toolkit/tools/imagecustomizer/docs)

Reply:
Will do, thanks.

Reply:
I have drafted one, please have a check. Thanks.